modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -492,6 +506,35 @@ private SearchContext _buildSearchContext(
 		return remoteAppEntries;
 	}
 
+	private RemoteAppEntry _startWorkflowInstance(
+			long userId, RemoteAppEntry remoteAppEntry)
+		throws PortalException {
+
+		ServiceContext serviceContext = new ServiceContext();
+
+		serviceContext.setAddGroupPermissions(true);
+		serviceContext.setAddGuestPermissions(true);
+		serviceContext.setCompanyId(remoteAppEntry.getCompanyId());
+		serviceContext.setScopeGroupId(WorkflowConstants.DEFAULT_GROUP_ID);
+		serviceContext.setUserId(userId);
+
+		Map<String, Serializable> workflowContext =
```

Comment:
If you are going to add just one value you can use Collections.singletonMap I think.

Reply:
True! Changing